.travis.yml

```diff
@@ -1,14 +1,13 @@
 # Travis CI configuration file for running tests
 language: python
+sudo: false
 python:
   - "2.7"
-before_install:
-  - pip install pep8
-  - pep8 --verbose
 install:
   - pip install --allow-all-external -r requirements.txt
-  - pip install coverage
+  - pip install --allow-all-external -r test_requirements.txt
```

Comment:
Nit: It's better to restrict this as `--allow-external some_problematic_package` than to use `--allow-all-external`, but I don't know how many packages were causing the issue here.